src/structures/GuildEmoji.js

```diff
@@ -90,6 +90,8 @@ class GuildEmoji extends Emoji {
   fetchAuthor() {
     if (this.managed) {
       return Promise.reject(new Error('EMOJI_MANAGED'));
+    } else if (!this.guild.me.permissions.has('MANAGE_EMOJIS')) {
```

Comment:
This should use `Permissions.FLAGS.MANAGE_EMOJIS` for consistency with other places.

Reply:
Will update -- I wasn't sure which to use, I found examples of usages both ways while looking around

Reply:
Actually, @kyranet -- which would be preferred, then?

```
this.guild.me.permissions.has(Permissio...etc
```
vs
```
this.guild.me.hasPermissions(Permission..etc
```

I'm finding both patterns used in various places

Reply:
`permissions.has`, `GuildMember#hasPermissions` is more useful for when you are checking permissions for the guild's owner, and a bot cannot own a guild (unless you're testing for GameBridge, which is a rare case):

https://github.com/discordjs/discord.js/blob/9b329a457c90eae43496f45f596005c8eb9fa662/src/structures/GuildMember.js#L228-L231

Reply:
makes sense! thanks, i've updated :)